frontend/talentsearch/src/js/components/GovernmentInfoForm/GovernmentInfoForm.tsx

```diff
@@ -434,7 +431,31 @@ export const GovInfoFormContainer: React.FunctionComponent = () => {
       return;
     }
     await handleUpdateUser(meId, data)
-      .then(() => {
+      .then((result) => {
+        if (result.isProfileComplete) {
```

Comment:
This code block is repeated exactly on each page.  Is there any way to factor this out to a single location to be more DRY?

Reply:
It makes sense. I will try.